index.js

```diff
@@ -45,8 +45,8 @@ module.exports.plugins = [
       { no: "Github", yes: "GitHub" },
       { no: "Javascript", yes: "JavaScript" },
       { no: "Node.JS", yes: "Node.js" },
-      { no: "Note that", yes: "<nothing>" },
-      { no: "note that", yes: "<nothing>" },
+      { no: "\\s+node.js", yes: "Node.js" },
```

Comment:
I'm pretty sure the plugin uses `\b` around the provided `no` string, so the `\\s+` should be unnecessary.

Reply:
It does not use `\b` but something more complicated: `(\\.|@[a-z0-9/-]*)?(${val.no})(\\.\\w)?`. Still, I'm pretty sure the `\\s+` is superfluous.

Reply:
(Also, the `\\s+` means this will miss things at the beginning of a line, so `\\b` is probably better if a delimiter is required.)

Reply:
Oh, also, the period needs to be escaped with `\\`.

Reply:
Hmm...I seem to be wrong about not requiring the word break.  `\\b` it is, I guess.

Reply:
So after all that:

```suggestion
      { no: "\\bnode\\.js", yes: "Node.js" },
```

Reply:
I might need to go back and see how my own plugin works again to make sure this is right. 😀

Reply:
Confirmed that the above suggestion seems to do what we want. Are there specific strings you're concerned about other than `@node.js`?